src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -214,8 +214,10 @@ namespace Foo
                 {
                     public class Car
                     {
-                        public int? Wheels { get; }
+                        public int? Drivers { get; }
+                        public int Wheels { get => 4; }
                         public bool IsRunning { get; set; }
+                        public bool Is4x4 { get => false; set { } }
```

Comment:
Can you please describe the ordering algorithm? Is it the exact same as in the old GenAPI? Asking as we want to preserve the current ordering for the reference source files in dotnet/runtime, i.e. [System.Runtime.cs](https://raw.githubusercontent.com/dotnet/runtime/main/src/libraries/System.Runtime/ref/System.Runtime.cs).

Reply:
I was confused that the tests (which weren't correctly sorted) didn't fail. I assume we don't yet have roundtripping tests?

Reply:
>I was confused that the tests (which weren't correctly sorted) didn't fail.
Nope, the tests for previous iteration did fail. That's why I made changes to expected code snippets in the last iteration.